Integrated-Windows-Auth-Flow/src/main/java/IntegratedWindowsAuthFlow.java

```diff
@@ -8,66 +8,122 @@
 import com.microsoft.aad.msal4j.PublicClientApplication;
 import com.microsoft.aad.msal4j.SilentParameters;
 
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.util.Collections;
+import java.util.Properties;
 import java.util.Set;
 
 public class IntegratedWindowsAuthFlow {
 
-    private final static String CLIENT_ID = "";
-    private final static String AUTHORITY = "https://login.microsoftonline.com/organizations/";
-    private final static Set<String> SCOPE = Collections.singleton("");
-    private final static String USER_NAME = "";
+    private static String authority;
+    private static Set<String> scope;
+    private static String clientId;
+    private static String username;
+    private static String accounts;
 
     public static void main(String args[]) throws Exception {
 
-        IAuthenticationResult result = acquireTokenIntegratedWindowsAuth();
-        System.out.println("Access token: " + result.accessToken());
-        System.out.println("Id token: " + result.idToken());
-        System.out.println("Account username: " + result.account().username());
-    }
-
-    private static IAuthenticationResult acquireTokenIntegratedWindowsAuth() throws Exception {
+        setUpSampleData();
 
-        // Load token cache from file and initialize token cache aspect. The token cache will have
-        // dummy data, so the acquireTokenSilently call will fail.
-        TokenCacheAspect tokenCacheAspect = new TokenCacheAspect("sample_cache.json");
+        TokenCacheAspect tokenCacheAspect = new TokenCacheAspect(accounts);
```

Comment:
i think it would be enough to use in-memory cache here - so no need to set TokenCacheAspect. Generally speaking  this aspect should be used if we want to use persistent cache